UI/IntroHero/index.js

```diff
@@ -1,24 +1,40 @@
 import React, { useState } from "react"
 import introStyles from "./introHero.module.css"
+import { navigate } from 'gatsby';
 import { Button } from '../Button';
 
-export function IntroHero({ onSubmit = console.log }) {
+export function IntroHero() {
    const [name, setName] = useState("")
 
    return (
-      <form className={introStyles.intro} onSubmit={e => onSubmit(e, { name })}>
+      <form className={introStyles.intro} onSubmit={e => {
+         e.preventDefault()
+         // Implementation of this function is an exercise for the reader.
+         navigate(
+            "/game",
+            {
+               state: { name },
+            }
+         );
+      }}>
```

Comment:
Think you missed this in a merge conflict?

Reply:
I think this is right isn't it as we dont have setPage anymore?

Reply:
fixed